docs/network/architecture.md

```diff
@@ -32,7 +33,7 @@ Each Etherlink block contains:
 The sequencer publishes each block in two ways:
 
 - It publishes them to EVM nodes, which can consider the transactions final as long as they trust that the sequencer will publish them to layer 1.
-- It publishes them to Tezos layer 1, which finalizes the transactions.
+- It publishes them to Tezos layer 1 via the DAL or via the Smart Rollup outbox, which finalizes the transactions.
```

Comment:
Why are you mentioning the outbox here?

Reply:
Removed mention of the outbox and just says "Tezos layer 1" instead. is it too much detail, or does the kernel not actually use the outbox?